style.css

```diff
@@ -0,0 +1,82 @@
+*{
+  font-family: sans-serif;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+  background-color: #232323;
+}
+
+.calculator {
+  background-color: #101010;
+  width: 370px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 4px solid #ffffff;
+}
+
+input {
```

Comment:
See if you can make this more specific. Maybe use a class to style the input field instead of styling the whole tag?